api.ts

```diff
@@ -238,9 +238,10 @@ export class CloudAPISDK extends Client {
      * Backing up a database
      * @param subscriptionId The id of the subscription 
      * @param databaseId The id of the database
+     * @param parameters The backup parameters (Optional)
```

Comment:
Why not call it region? also for `BackupRegion`

Reply:
There is already a type Region in the SDK, so i've renamed this type `RegionName`